players.md

```diff
@@ -0,0 +1,85 @@
+# Player Differences
+
+Swiftfin offers two player options: the default Swiftfin player and Native (AVKit). The Swiftfin team recommends using the Swiftfin player for optimal compatibility and features, though the Native player is also available for certain cases that benefit from Apple's native capabilities. All video, audio, and subtitle formats listed are supported for direct playback. If transcoding is enabled on your server, any unsupported formats will be converted automatically.
```

Comment:
Language here is a bit imprecise because most things on Native are going to be repackaged in HLS even if video/audio codec are supported for DirectPlay, given container limitations (e.g. anything in mkv). This means that most things, disregarding subtitles, not in MP4 container will be remuxed or DirectStreamed if video codec is supported and transcoded otherwise.

Reply:
I added a container section, do you think that suffices? Here is my up-to-date version:

https://github.com/JPKribs/Swiftfin/blob/playerDocumentation/players.md

Reply:
```suggestion
Swiftfin offers two player options: the default Swiftfin player and Native (AVKit). The Swiftfin team recommends using the Swiftfin player for optimal compatibility and features, though the Native player is also available for certain cases that benefit from Apple's native capabilities. All video, audio, and subtitle formats listed are supported for direct playback but might be repackaged based on container support. If transcoding is enabled on your server, any unsupported formats will be converted automatically.
```
I might consider adding that addendum just to be clear, but ultimately it doesn't make too much of a difference.